docs/indexing.md

```diff
@@ -221,7 +221,7 @@ CTRL+b d (to detach from tmux)
 Then swap the rebuild collection to the production alias.
 
 - Run the cap task on pul_solr to swap the aliases
-- Make sure you use the right values for PRODUCTION and REBUILD. For example if you just built the full index on catalog-production2 and are swapping it into production do:
+- Make sure you use the right values for PRODUCTION and REBUILD. Instructions for determining the correct values are [in the cap task](https://github.com/pulibrary/pul_solr/blob/e965de4516c8b693c2dd384ddd14371415990d78/config/deploy.rb#L99-L101).  For example if you just built the full index on catalog-production2 and are swapping it into production do:
```

Comment:
@sandbergja can you please change the path to: https://github.com/pulibrary/pul_solr/blob/main/config/deploy.rb#L99-L101 so that it does not include the commit hash?

Reply:
Good idea!  I made that change.